Bottom: ae9f4dbeacfeb5bd98658f9a56935c7e8a41566b
Top:    70adcce0b3da9b172fe9b1dd4f908931a9c06ff3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 18:41:15 +0000

find.c: Fix flag passed to ListStart() in DRCFind().

We should have set the flag before calling ListStart(), as it marks
the first item with that flag.


---

diff --git a/src/find.c b/src/find.c
index 9198df4..68b65d7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3360,8 +3360,8 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
           ListStart (What, ptr1, ptr2, ptr3, flag);
           DoIt (flag, true, true);
           DumpList ();
-          ListStart (What, ptr1, ptr2, ptr3, flag);
           flag = FOUNDFLAG;
+          ListStart (What, ptr1, ptr2, ptr3, flag);
           Bloat = 0;
           drc = true;
           DoIt (flag, true, true);
